assignment4.Rmd

```diff
@@ -111,6 +116,49 @@ biplot(pca)
 
 #Calculate values for each student that represent these your composite variables and then create a new correlogram showing their relationship to mean_correct.
 
+D5<- as.matrix(D2) %*% pca$rotation
+D6<- cbind(D5, as.data.frame(D1$mean_correct))
+COR3 <- cor(D6)
+
+corrplot(COR3, order="AOE", method="circle", tl.pos="lt", type="upper",        
+ tl.col="black", tl.cex=0.6, tl.srt=45, 
+         addCoef.col="black", addCoefasPercent = TRUE,
+         sig.level=0.50, insig = "blank")
 ```
 # Part III  
 ## Also in this repository is a data set and codebook from Rod Martin, Patricia Puhlik-Doris, Gwen Larsen, Jeanette Gray, Kelly Weir at the University of Western Ontario about people's sense of humor. Can you perform a PCA on this data?
+```{r}
+DF <- read.csv("humor_data.csv", header=TRUE)
+DF1 <- DF1[1:32]
+COR4 <- cor(DF1)
+corrplot(COR4, order="AOE", method="circle", tl.pos="lt", type="upper",        
+tl.col="black", tl.cex=0.6, tl.srt=45, 
+        addCoef.col="black", addCoefasPercent = TRUE,
+        sig.level=0.50, insig = "blank")
+DF2 <- scale(DF1, center = TRUE,scale=TRUE)
+pca2 <- prcomp(DF2, scale = TRUE)
+pca2$sdev
+pca2$sdev^2
+summary(pca2)
+plot(pca2, type = "lines")
+biplot(pca2)
+DF3<-as.data.frame(pca2$x)
+DF4<-subset(DF3,select=c(1:6))
+DF5<-cbind(DF4,as.data.frame(DF[,33:36]))
+DF6<-scale(DF5,center=TRUE)
+COR4<-cor(DF6)
+ corrplot(COR4, order="AOE", method="circle", tl.pos="lt", type="upper",        
+ tl.col="black", tl.cex=0.6, tl.srt=45, 
+         addCoef.col="black", addCoefasPercent = TRUE,
+         sig.level=0.50, insig = "blank")
+
```

Comment:
Great coding work! Can you interpret the output though? Did you replicate the findings of the researchers?